<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseGenerator</class>
 <widget class="QDialog" name="DatabaseGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>561</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Font Texture Generator</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_ModeEdit">
         <property name="text">
          <string>Edit Existing .fte</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">BG_Mode</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_ModeNew">
         <property name="text">
          <string>Import New Font</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">BG_Mode</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="PB_Import">
         <property name="text">
          <string>Import .fte</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_Export">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Export .fte and Textures</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>304</width>
          <height>495</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Buttons</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Texture:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="LE_ButtonTexture">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="PB_Texture">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="GL_Button">
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>A</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Width</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Height</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Y</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Y</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>B</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>RT</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>LT</string>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Back</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>RB</string>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>L-Stick</string>
                </property>
               </widget>
              </item>
              <item row="12" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>R-Stick</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>LB</string>
                </property>
               </widget>
              </item>
              <item row="13" column="0">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>D-Pad</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Characters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Font:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="LE_Font">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="PB_Font">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_20">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SB_FontSize">
                <property name="minimum">
                 <number>12</number>
                </property>
                <property name="maximum">
                 <number>48</number>
                </property>
                <property name="value">
                 <number>21</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Spacing:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SB_Spacing">
                <property name="maximum">
                 <number>50</number>
                </property>
                <property name="value">
                 <number>2</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_22">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>OffsetXY:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SB_OffsetX">
                <property name="minimum">
                 <number>-50</number>
                </property>
                <property name="maximum">
                 <number>50</number>
                </property>
                <property name="value">
                 <number>-3</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SB_OffsetY">
                <property name="minimum">
                 <number>-50</number>
                </property>
                <property name="maximum">
                 <number>50</number>
                </property>
                <property name="value">
                 <number>20</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Character List:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="TE_Characters"/>
            </item>
            <item>
             <widget class="QPushButton" name="PB_ExportDatabase">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Export fcoDatabase.txt</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="RB_Button">
         <property name="text">
          <string>Button Texture</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">BG_Texture</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_Font">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Font Textures</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">BG_Texture</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SB_FontIndex">
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ZoomGraphicsView" name="GV_Preview">
       <property name="minimumSize">
        <size>
         <width>520</width>
         <height>520</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>512</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>zoomgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="BG_Mode"/>
  <buttongroup name="BG_Texture"/>
 </buttongroups>
</ui>
